app/renderer/components/styles/global.js

```diff
@@ -134,8 +134,6 @@ const globalStyles = {
     switchNubTopMargin: '2px',
     switchNubLeftMargin: '2px',
     switchNubRightMargin: '2px',
-    buttonHeight: '25px',
-    buttonWidth: '25px',
```

Comment:
Please search for `buttonHeight` and `buttonWidth` in case where it be used on anywhere.